### PR TITLE
WCMS-13449 Update dataset search on browser back button

### PR DESCRIPTION
**Pull request by brdunfield**

### CODE REVIEW

File: src/templates/DatasetSearch/index.jsx [hunk 1]
```diff
@@ -104,6 +104,13 @@ const DatasetSearch = ({
     }
   }, [fulltext]);
 
+  useEffect(() => {
+    // update search on browser back button press
+    const updatedFullText = location.search.indexOf("=") !== -1 ? location.search.split("=")[1] : "";
```
Comment (dgading) [inline src/templates/DatasetSearch/index.jsx:109]: We need a check to make sure this is actually the fulltext search param. I clicked on page 5 without doing a search and it filled the url bar in the browser and the fulltext search input with `fulltext=5%25252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252525252526fulltext`

**Review by dgading — CHANGES_REQUESTED** [review]
The fulltext feature works well, but it seems like the other input actions that do automatic submits cause a loop to happen.

File: src/templates/DatasetSearch/index.jsx (hunk 1 above)
Comment (brdunfield) [reply src/templates/DatasetSearch/index.jsx]: Great catch! It should be isolated now. Tested clicking around a few pages and filters and it seems to function correctly now for me.

**Comment by brdunfield** [comment]
This is so old, I'm closing it. 🧹

**Pull request closed without merging**